oasislmf/computation/run/platform.py

```diff
@@ -135,24 +116,19 @@ def print_endpoint(self, attr, items):
     def select_id(self, msg, valid_ids):
         while True:
             try:
-                value = int(input(f'Select {msg} ID: '))
+                value = input(f'Select {msg} ID: ')
```

Comment:
I think `input` already returns str, but since you have it later, let's add it for safety.
I'd put it here, though:
```suggestion
                value = str(input(f'Select {msg} ID: '))
```